ddsc/cmdparser.py

```diff
@@ -44,6 +44,33 @@ def add_project_name_arg(arg_parser, required=True, help_text="Name of the remot
                             required=required)
 
 
+def add_project_id_arg(arg_parser, required=True, help_text="Name of the remote project to manage."):
+    """
+    Adds project_name parameter to a parser.
+    :param arg_parser: ArgumentParser parser to add this argument to.
+    :param help_text: str label displayed in usage
+    """
+    arg_parser.add_argument("-i",
```

Comment:
Could we add a long-form for this like `--project-id`? While `-i` is short and makes sense, I think `--project-id` would be more readable when documenting usage.